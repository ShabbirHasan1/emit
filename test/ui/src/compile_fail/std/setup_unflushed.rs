#![deny(warnings)]

fn main() {
    let rt = emit::setup().init();
}
