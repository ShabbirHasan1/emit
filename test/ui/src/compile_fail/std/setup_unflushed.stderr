error: unused variable: `rt`
 --> src/compile_fail/std/setup_unflushed.rs:4:9
  |
4 |     let rt = emit::setup().init();
  |         ^^ help: if this is intentional, prefix it with an underscore: `_rt`
  |
note: the lint level is defined here
 --> src/compile_fail/std/setup_unflushed.rs:1:9
  |
1 | #![deny(warnings)]
  |         ^^^^^^^^
  = note: `#[deny(unused_variables)]` implied by `#[deny(warnings)]`
